Resources/Prototypes/Catalog/Fills/Backpacks/StarterGear/backpack.yml

```
@@ -48,16 +48,16 @@
       - id: Flash
       #- name: StationCharter
       #- name: TelescopicBaton
+
 - type: entity
   noSpawn: true
-  parent: ClothingBackpackEngineering
+  parent: ClothingBackpackCE #Corvax-Resprite
```

Просто скопируй то как я показал, с **пробелами**
```suggestion
  parent: ClothingBackpackCE # Corvax-Resprite
```

---

Вроде готово